test/helpers/mock-runner.js

```diff
@@ -1,52 +0,0 @@
-'use strict';
-
-var EventEmitter = require('events').EventEmitter;
-var util = require('util');
-
-// mock test runner
-function Runner() {
```

Comment:
These mocks are no longer needed since we now instantiate real Runners, Suites, and Tests.